content/en/docs/tasks/security/authentication/authn-policy/index.md

```diff
@@ -303,7 +303,7 @@ EOF
 {{< /text >}}
 
 1. The port value in the peer authentication policy is the container's port. The value the destination rule is the service's port.
-1. You can only use `portLevelMtls` if the port is bound to a service. Istio ignores it otherwise.
+2. You can only use `portLevelMtls` if the port is bound to a service. Istio ignores it otherwise.
```

Comment:
The lines should all be numbered with `1.` and not increment each section.

Reply:
It's also good that the linter has identified these as well.

Reply:
thx for check，I correct the line number and indents.